pandas/tests/frame/methods/test_align.py

```diff
@@ -392,27 +392,57 @@ def test_missing_axis_specification_exception(self):
         with pytest.raises(ValueError, match=r"axis=0 or 1"):
             df.align(series)
 
-    def _check_align(self, a, b, axis, fill_axis, how, method, limit=None):
+    @pytest.mark.parametrize("method", ["pad", "bfill"])
+    @pytest.mark.parametrize("axis", [0, 1, None])
+    @pytest.mark.parametrize("fill_axis", [0, 1])
+    @pytest.mark.parametrize("how", ["inner", "outer", "left", "right"])
+    @pytest.mark.parametrize(
+        "left_slice",
+        [
+            [slice(4), slice(10)],
+            [slice(0), slice(0)],
+        ],
+    )
+    @pytest.mark.parametrize(
+        "right_slice",
+        [
+            [slice(2, None), slice(6, None)],
+            [slice(0), slice(0)],
+        ],
+    )
+    def test_align_fill_method(
+        self, how, method, axis, fill_axis, float_frame, left_slice, right_slice
+    ):
+        limit = 1
```

Comment:
Are we missing a case where `limit = None` from the original `_check_align_fill` implementation?

Reply:
Good catch! Add a `parametrize` over `limit = [None, 1]`